### DISCUSSION TITLE
question about tab control

### DISCUSSION THREAD

**Discussion by zhefenfei** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2992] [Q&A]
tabcontrol add close icon . icon X  is close funcition .when the tab item is not selected. close is working. or not ,the close can not receive mouse event  just like some thing upon the icon..
![image](https://user-images.githubusercontent.com/30534067/206980562-ebca63e9-c74a-46b8-8e87-f2fe4e5831ab.png)
mouse move to not selected tab close icon.
![image](https://user-images.githubusercontent.com/30534067/206980702-a4518792-a8c9-46a3-8532-93291a02a81f.png)
mouse move to selected tab close icon ,no event recieved.

**Comment by zhefenfei** [comment]
i want to implements no matter what selected state or not , i can close the tab...

**Comment by nicolaihenriksen** [comment]
@zhefenfei There is an open PR which should fix this issue: #2938. However, there are some merge conflicts that need to be resolved, and the author has not responded in a while. Perhaps @Keboo can spend 2 mins on fixing the branch (if the author allowed him to edit it)?

**Comment by nicolaihenriksen** [comment]
This has been possible since #2938 merged.
